packages/api/src/store/model.js

```diff
@@ -188,7 +188,7 @@ export default class Model {
     return responses;
   }
 
-  close() {
+  async close() {
     this.db.close();
```

Comment:
Might want to add an `await` or `return` here?

Reply:
yes, added return statement.